src/tools/rbd/action/ObjectMapCheck.cc

```diff
@@ -0,0 +1,74 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Well, right now there is a 1:1 correspondence between tool actions and the source files that implement them (cf. ObjectMapUpdateRequest.cc). ObjectMap.cc contains the library tooling and this seemed like a user-level operation like the others in operations/. I'll change it if you want, though.